README.md

```diff
@@ -148,4 +159,25 @@ add-namespace:
 # List of tasks to run when running `upgrade-code doctor`
 doctorTasks:
   SilverStripe\Dev\CleanupInstall: src/Dev/CleanupInstall.php
+warnings:
+  classes:
+    MyClass:
+      message: 'MyClass has been removed'
+      url: 'http://my-domain/upgrade-instructions'
+  methods:
+    'MyClass->myInstanceMethod()':
+      message: 'Use otherMethod() instead'
+    'MyClass::myStaticMethod()':
+      message: 'Use otherMethod instead'
+  props:
+    'MyClass->myInstanceProp'
+      message: 'Use otherProp instead'
+    'MyClass::myStaticProp'
+      message: 'Use otherProp instead'
+  functions:
+    'myFunction()':
+      message: 'Use otherFunction() instead'
+  constants:
+    'MY_CONSTANT':
+      message: 'Use OTHER_CONSTANT instead'
```

Comment:
Clarify how to specify class constants.

Reply:
I don't think you can at the moment. I can't think of a case where that would apply in the 4.x upgrade, so didn't implement it.

Reply:
Can you please document that this only works with non-class constants then? Otherwise someone might be confused if it breaks.

Reply:
I've implemented it now :)